### ISSUE TITLE
Server Crash Bug is back?

### ISSUE THREAD

**Issue by joemisfit** [SmartlyDressedGames/Unturned-3.x-Community#705]
Hello @SDGNelson 

As we've reported previously the cheat clients that have crashed servers via local packet pushing to serverside or other clients, I believe it may be back due to the fact that all of the staff were force crashed off of the server at the exact same time, and we banned a 2 people which were suspicious due to the time frame they entered the server, and what their profiles entailed, surely enough the staff were able to rejoin. The first person didn't allow anyone to rejoin, but the second did. This was after a short situation in the server regarding untrue claims and a permaban. 

Basically each of the staff members games completely crashed. Which is what normally happens when packets are being pushed client side. I've also had reports from other server owners that this has been happening to them recently as well.

If this could be looked into, that would be greatly appreciated, so we don't have another Dead Unturned Community, due to players being scared of playing servers because of cheaters.

Thanks!

**Comment by Coopyy** [comment]
no

**Comment by AdovinOFFICIAL** [comment]
what this guy said ^

**Comment by SDGNelson** [comment]
Thankfully I received a report this weekend of what I believe is the vulnerability, and it should be patched in the update in the next couple days. Will leave this open so we can see how it goes after the update

**Comment by Coopyy** [comment]
no

**Comment by AdovinOFFICIAL** [comment]
what this guy said ^

**Comment by joemisfit** [comment]
Having fun trolling my post @Coopyy @AdovinOFFICIAL ?

**Comment by Coopyy** [comment]
why did you put my name in the update notes, saying "no" fixed your problems?

**Comment by AdovinOFFICIAL** [comment]
what this guy said ^

**Comment by Coopyy** [comment]
thanks for b1g shoutout nolson, really an honor

**Comment by AdovinOFFICIAL** [comment]
yes thank you for helping him patch this absurd vulnerability. by saying "no" you really helped him out.

**Comment by SDGNelson** [comment]
Ah, from your report email I thought you wanted a credit to get back at your cheater cohorts. I'll anonymize that line in the update notes.

**Comment by Coopyy** [comment]
good meme boss

**Comment by AdovinOFFICIAL** [comment]
yeah nice bait

**Comment by Coopyy** [comment]
also i didnt say to remove my name, i want my big fame back :[
